docker/php-fpm/Dockerfile

```diff
@@ -6,5 +6,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 # Install selected extensions, git and other stuff
 RUN apt-get update \
-    && apt-get -y --no-install-recommends install php7.2-phpdbg php7.2-mysql php7.2-gd php-imagick php7.2-odbc php-xdebug php7.2-xmlrpc mysql-client unzip git gettext \
+    && apt-get -y --no-install-recommends install build-essential php-pear php7.2-dev php7.2-phpdbg php7.2-mysql php7.2-gd php-imagick php7.2-odbc php-xdebug php7.2-xmlrpc mysql-client unzip git gettext \
```

Comment:
Wofür werden die Module benötigt?

Reply:
Damit Xdebug auf Version 2.9.0 geupdated werden kann und wir den Fix für den Slowdown bei der Code Coverage haben.
Pear ist dafür da damit wir xdebug via pecl runterladen können, php7.2-dev enthält die Build-Header damit Xdebug zur entsprechenden Version gebuildet werden kann und build-essentials enthält die Tools zum Builden wie z.B. "make".

Reply:
Im Vergleich:
![image](https://user-images.githubusercontent.com/3511228/70934871-3a380600-203f-11ea-9f2e-469876ab754f.png)

24:22 Minuten Aufgrund des Slowdowns von Xdebug ab Version 2.8

zu:
![image](https://user-images.githubusercontent.com/3511228/70934899-4a4fe580-203f-11ea-9246-d0b9503e0b05.png)

11:30 mit dem kürzlich veröffentlichen Fix in Version 2.9